meilisearch-lib/src/tasks/error.rs

```diff
@@ -28,6 +30,7 @@ impl ErrorCode for TaskError {
     fn error_code(&self) -> Code {
         match self {
             TaskError::UnexistingTask(_) => Code::TaskNotFound,
+            TaskError::InvalidTask(_) => Code::TaskNotFound,
```

Comment:
Do we want to return a 404 or more a 400 for malformed query?

Reply:
Since you are requesting a task that will never exist, it seems consistent and ok to return a 404 IMO.

Reply:
Thanks for the reviews. I honestly didn't give this any thought to the error code. Looking at `InvalidIndexUid` and `InvalidDocumentId` errors I feel 400 is better, with new error `Code::InvalidTaskId`. If you guys agree I will make that change.

Reply:
I honestly prefer that we continue to be consistent with the `InvalidIndexUid` and `InvalidDocumentId` errors, thus returning a 400. What do you think @gmourier?

Reply:
FYI @chammika-become, @gmourier and @Kerollmops are on Holiday for a few weeks, so they will answer when they come back 😄